### PR TITLE
Adding many new strings

### PR DESCRIPTION
**Pull request by AirwavesMan**
- adding many new german strings
- corrected and changed a few because many were too long

- changed the safe description because it was a way too long even 1051 half of the description was not visible. It is a better now.

- adding new localizations for some building items. Ive changed the name of some buldings like Small Wall to Wood Wall to get a better consistency with the buildable items. Even the same in german. 

- remove options are now listet in the stringtable

- almost every english string should have an german string too now

- adding the Skin_Sniper1W_DZN string for better compatibility with namalsk @ndavalos

### CODE REVIEW

**Comment by ndavalos** [comment]
Thanks for the namalsk ghillie addition, since that string is no longer used in epoch anymore and is specific to namalsk, might rename that ghillie to be a "winter ghillie" or something, just a thought :)

**Comment by AirwavesMan** [comment]
Ohhh sry, after 2 times reviewing still so many things I missed. I left the Wood Stairs Rail in my first commit. I added it now.

File: SQF/dayz_code/Configs/CfgVehicles/DZE/Prop_Defs.hpp [hunk 1]
```diff
@@ -370,7 +370,7 @@ class Supply_Crate_DZE: MiningItems
 
 	mapSize = 2;
 	armor = 40;
-	displayName = "Supply Crate";
+	displayName = $STR_AMMO_SUPPLY_WRECK;
```
Comment (ebaydayz) [inline SQF/dayz_code/Configs/CfgVehicles/DZE/Prop_Defs.hpp]: You forgot to change this one.

File: SQF/dayz_code/Configs/CfgVehicles/DZE/Doors.hpp [hunk 2]
```diff
@@ -204,73 +204,73 @@ class CinderWallDoorLocked_DZ_Base: DZE_Housebase {
...
+	displayName = $STR_WOOD_WALL_WRECK;
 	removeoutput[] = {{"PartWoodPlywood",{0,1}},{"PartWoodLumber",{0,1}}};
 };
 
 class Land_wood_wreck_half : ruins	{
 	scope = protected;
 	model = "\z\addons\dayz_epoch\models\wood_wreck_half.p3d";
-	displayName = "Wood Floor 1/2 ruins";
+	displayName = $STR_ WOOD_FLOOR_WRECK;
 	removeoutput[] = {{"PartWoodPlywood",{0,1}},{"PartWoodLumber",{0,1}}};
```
Comment (ebaydayz) [inline SQF/dayz_code/Configs/CfgVehicles/DZE/Doors.hpp]: You added an extra space by accident on these.

File: SQF/dayz_code/stringtable.xml [hunk 3]
```diff
@@ -15271,153 +15320,207 @@
...
 		</Key>
 		<Key ID="STR_EPOCH_WOODSTAIRSWITHSUPPORTS">
 			<English>Wooden Stairs w/ supports</English>
+			<German>Holztreppe (verstärkt)</German>			
 		</Key>
 		<Key ID="STR_EPOCH_WOODSTAIRSWITHSUPPORTS_DESC">
 			<English>Wooden stairs with supports.</English>
+			<German>Holztreppe mit verstärkten Balken.</German>	
+		</Key>
+		<Key ID="$STR_EPOCH_WOODSTAIRSRAILS">
```
Comment (ebaydayz) [inline SQF/dayz_code/stringtable.xml]: You added a $ in the name by accident.

**Comment by AirwavesMan** [comment]
I hope it is good now

**Pull request merged**